pkg/scheduler/factory.go

```diff
@@ -228,11 +223,16 @@ func (c *Configurator) createFromProvider(providerName string) (*Scheduler, erro
 		return nil, fmt.Errorf("algorithm provider %q is not registered", providerName)
 	}
 
+	pluginConfig := []schedulerapi.PluginConfig{
+		c.volumeBindingPluginConfig(),
+	}
+
 	for i := range c.profiles {
 		prof := &c.profiles[i]
 		plugins := &schedulerapi.Plugins{}
 		plugins.Append(defaultPlugins)
 		plugins.Apply(prof.Plugins)
+		prof.PluginConfig = append(prof.PluginConfig, pluginConfig...)
```

Comment:
If a user provides a configuration for the plugin, we shouldn't configure it. Also add a test case.